### ISSUE TITLE
Too small increment

### ISSUE THREAD

**Issue by PaulBlanche** [haikarainen/light#98]
My system seems to handle brigthness by increment of 1/12
```
> light -S 100
> light -Gr
12
```
This means that 1 increment is 8.33%. If i try to increase the brightness by 5%, nothing happens as expected. But multiple increments of 5% does nothing as well :
```
> light -Sr 1
> light -Gr
1
> light -A 5
> light -Gr
1
> light -A 5
> light -Gr
1
```
After the second increment i expected the raw value to change to `2` (since two increment of 5% makes a 10%, more than the needed 8.33%).

It seems xbacklight handles things differently, by rounding up to the smallest increment (so `xbacklight -inc 5` actually does increase the luminosity by 8.33%). It is not what is expected either, but at least brightness did increase.

I don't know if it is the intended behavior, but if it is, maybe the readme could be more explicit ?

**Comment by haikarainen** [comment]
This is basically intended behaviour. I think it was even stated in the readme at some point. Anyway, for very shitty controllers I recommend using raw values instead of percent, as they are much more accurate for lowresolutuöion controllers.

Also, please verify that that is the best controller you have. Auto should go to the controller with highest resolution, but this might be indicative of a bug in the auto resolver.

**Comment by PaulBlanche** [comment]
I checked, it's the best controller i have.
It's a shame it's not working that ways, because in my opinion it undermines the usefulness of values in percentage. But if it is the intended behavior, i'll update my scripts.

**Comment by haikarainen** [comment]
Im sorry, but the only thing undermining the usefulness of dividing your controllers backlight-levels by 100 levels is the fact that it only provides 12 levels.  :p

Using percentage with your controllers would just make it highly inaccurate instead, even if it would "feel" better to mask the bad precision with the % symbol.

**Comment by PaulBlanche** [comment]
> Im sorry, but the only thing undermining the usefulness of dividing your controllers backlight-levels by 100 levels is the fact that it only provides 12 levels. :p

I understand, but if i want to use percentage, it can also be to abstract away the resolution of a controller (having the same scale for different controllers), however shitty it might be ^^. If i have to take care of the resolution of the controller when abstracting it away, then it's a leaky abstraction.

In my case, i have two laptops, and i want to maintain the same script to handle brightness on both. I saw that i could use raw value, but on different laptop i expected the range to be different, so i settled on percentage to abstract away the difference between the two laptop. On the first one i used `light -A 5` which was working. I started using the second laptop (the one with only 12 levels), and the script was not working anymore. This was surprising to me, since nothing nowhere explained why it did not work. With no further explanations on the resolution of controllers, i wrongly expected percentage to work that way.

I'm not arguing that it should be the desired behavior, or event that the current behavior is a bad one, it's just that it is a surprising behavior with the information given. Then again it does not takes a special brain to try raw values as i did so ...
